net.casimirlab.frigoligo.metainfo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.casimirlab.frigoligo</id>
```

Comment:
`casimirlab.net` doesn't see to exist as a domain

yet:

> [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
I can't reach it still.

Also do you have control over the whole domain or only the subdomain?

Reply:
https://casimirlab.net/ needs to be reachable HTTP 200 you aren't using the subdomain in the app ID.

Reply:
I have the control of the whole domain but I only use subdomains for my services.

I have set up a static server that return a 204 for https://casimirlab.net and a well known file for the subsequent verification at https://casimirlab.net/.well-known/org.flathub.VerifiedApps.txt

Does that work for you?

Reply:
yes